SCAutolib/models/CA.py

```diff
@@ -1,12 +1,18 @@
 """
-This module implements classes that represents Certification Authorities (CA).
+This module implements classes that represent Certification Authorities (CA)
```

Comment:
```suggestion
This module implements classes that represent Certificate Authorities (CA)
```
https://en.wikipedia.org/wiki/Certificate_authority